mpqp/tools/maths.py

```diff
@@ -161,7 +161,7 @@ def closest_unitary(matrix: Matrix):
 
 
 @typechecked
-def cos(angle: Expr | Real) -> sp.Expr | float:
+def cos(angle: Expr | Real | float) -> sp.Expr | float:
```

Comment:
In principle, no need to add `float` here, as it is "contained" in `Real`

Reply:
yes, but i had typcheck issues